test/queries/searchResultQuery.test.ts

```diff
@@ -243,8 +242,8 @@ describe('SearchResultQuery', () => {
     });
   });
 
-  describe('given all valid filters', () => {
-    describe('given user has search results', () => {
+  describe('given special characters in url params', () => {
+    describe('given valid params for special characters in url filter', () => {
```

Comment:
```suggestion
    describe('given there are special characters filters', () => {
```

What do you think? 👀